codeclimate/resource_repository.go

```diff
@@ -78,3 +73,17 @@ func resourceRepositoryCreateForOrganization(d *schema.ResourceData, client inte
 
 	return err
 }
+
+func resourceRepositoryDelete(d *schema.ResourceData, client interface{}) error {
+	repositoryID := d.Get("codeclimate_id").(string)
+
+	c := client.(*codeclimateclient.Client)
+	err := c.DeleteOrganizationRepository(repositoryID)
+	if err != nil {
+		return err
+	}
+
+	d.SetId("")
```

Comment:
Why is this need if we are deleting? 🤔

Reply:
From my reading, it is done automatically but you can be safe by adding it.